Block/Js.php

```diff
@@ -458,11 +458,18 @@ public function isBoltProductPage()
      */
     public function getButtonCssStyles()
     {
+        $style = "";
+
+        $toggleCheckout = $this->configHelper->getToggleCheckout();
+        if ($toggleCheckout && $toggleCheckout->active) {
+            $style .= 'display:none;';
+        }
```

Comment:
not sure if we stick carefully to affirmative naming but not sure about the toggle being `active` to hide the button. Thoughts @matt-thomason @vitaliyreznikov @danac-gs ?

Reply:
$toggleCheckout->active is a static parameter.
So config can be like
```
     {
       "toggleCheckout": {
         "active": true,
         "magentoButtons": [                       
           "#top-cart-btn-checkout",
           "button[data-role=proceed-to-checkout]"
         ],
         "showElementsOnLoad": [                      
           ".checkout-methods-items",                
           ".block-minicart .block-content > .actions > .primary"
         ]
       }
     } 
```

If we change `active:true` to `active: false` it means the same as if we delete all config.
This is like things work in our codebase and I leave it here to be consistent.

Reply:
Okay, if it's internally consistent then looks good to me